blog/migrations/0019_migrate_image_data_to_custom_image_model.py

```diff
@@ -0,0 +1,50 @@
+from django.conf import settings
+from django.db import migrations
+
+
+def move_existing_image_data_to_custom_model(apps, _schema_editor):
+    wagtail_image_model = apps.get_model("wagtailimages", "image")
+    custom_image_model = apps.get_model("blog", "customimage")
+    custom_image_model.objects.bulk_create([
+        custom_image_model(
+            title=image.title,
+            file=image.file,
+            width=image.width,
+            height=image.height,
+            created_at=image.created_at,
+            uploaded_by_user=image.uploaded_by_user,
+            tags=image.tags,
+            focal_point_x=image.focal_point_x,
+            focal_point_y=image.focal_point_y,
+            focal_point_width=image.focal_point_width,
+            focal_point_height=image.focal_point_height,
+            file_size=image.file_size,
+            file_hash=image.file_hash,
+        ) for image in wagtail_image_model.objects.all().order_by("pk")
```

Comment:
**all()** should return all objects ordered by PK by default.
But when I used it alone, a bug happened and the wrong image was placed in the article's body!
I didn't have any more time to properly test it though, so I just added an explicit **order_by()**.